docs/languages/cpp.md

```diff
@@ -21,95 +21,112 @@ C/C++ support for Visual Studio Code is provided by a [Microsoft C/C++ extension
 1. Search for `'C++'`.
 1. Select **Install**.
 
-![Search for c++ in the Extensions view](images/cpp/search-cpp-extension.png)
+![C/C++ extension](/docs/cpp/images/cpp/cpp-extension.png)
```

Comment:
While reaching into another folder works on GitHub preview, I'm not sure it will render correctly on code.visualstudio.com when the build does the image path preprocessing.
It is also better to duplicate the required images in each TOC root folder in case topics are refactored or moved.

Reply:
fixed